TurkishFilter/sections/css_extended.txt

```diff
@@ -6,6 +6,7 @@ vizyonfilmizle.tv#$#.fancybox-lock body[-ext-has="> div"] { overflow: visible!im
 vizyonfilmizle.tv#$#html[-ext-has="> body"] { overflow: visible!important; }
 !
 fullkurulum.com#?#.entry-content > center:has(> a[href^="http://bc.vc/"] > img)
+torntube.com#?#.wrap > h1[style]:has(> span:contains(TELEFONDA))
```

Comment:
Rule with red text is enough, similar sites do not have any red text except the ads.